<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceDialog</class>
 <widget class="QDialog" name="ReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>199</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>472</width>
    <height>199</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>472</width>
    <height>199</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Replace</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_text_12">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_67">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>210</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>First occurrence</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_68">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>160</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>All document</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Find_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Replaced:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_69">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>240</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>All following occurences</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_23">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>134</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Case-sensitive</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Find_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Replacing:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_text_13">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>91</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ReplaceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ReplaceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
